README.md

```diff
@@ -114,6 +115,19 @@ Hurray! You successfully made a contribution! 🎉
 
 ## <ins>Hacktoberfest community<ins>
 
+### **Contents**
+
+|          |         |      |      |     |          |
+| ---------| ------- | ---- | ---- | --- | -------- |
+| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
+| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
+| [C](#c)  | [I](#i) | [O](#o) | [U](#u) |
+| [D](#d)  | [J](#j) | [P](#p) | [V](#v) |
+| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) |
+| [F](#f)  | [L](#l) | [R](#r) | [X](#x) |
+
```

Comment:
Maybe, the empty column at the end can be removed?

|          |         |      |      |     |       
| ---------| ------- | ---- | ---- | --- |
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------

```suggestion
|          |         |      |      |     |       
| ---------| ------- | ---- | ---- | --- |
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------
```

Reply:
exactly what I was suggesting

Reply:
> 
> 
> Maybe, the empty column at the end can be removed?
> 				
> [A](#a) 	[G](#g) 	[M](#m) 	[S](#s) 	[Y](#y)
> [B](#b) 	[H](#h) 	[N](#n) 	[T](#t) 	[Z](#z)
> [C](#c) 	[I](#i) 	[O](#o) 	[U](#u) 	
> [D](#d) 	[J](#j) 	[P](#p) 	[V](#v) 	
> [E](#e) 	[K](#k) 	[Q](#q) 	[W](#w) 	
> [F](#f) 	[L](#l) 	[R](#r) 	[X](#x)

### I have tried, this happens @Vyvy-vi @adityaraute :

![image](https://user-images.githubusercontent.com/70807500/135708728-8c6498e4-5ba4-45de-ab52-7feb4064f8ad.png)

thanks

Reply:
The included table in my suggestion seems to be rendering alright and without the extra field

I think you misinterpreted my comment... I was suggesting the "column" (vertical field) rather than the line at bottom

Reply:
```
|          |         |      |      |     |       
| ---------| ------- | ---- | ---- | --- |
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------
```

|          |         |      |      |     |       
| ---------| ------- | ---- | ---- | --- |
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------

Reply:
@Vyvy-vi now I see. Sorry for that!

Reply:
Do you mean these:
![image](https://user-images.githubusercontent.com/70807500/135712291-f0381ee4-48ef-45c4-9599-56dc38d7f464.png)

Reply:
Also, if we want to remove the row on top, this should work -
```
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| ---------| ------- | ---- | ---- | --- |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------
```
| [A](#a)  | [G](#g) | [M](#m) | [S](#s) | [Y](#y) |
| ---------| ------- | ---- | ---- | --- |
| [B](#b)  | [H](#h) | [N](#n) | [T](#t) | [Z](#z) |
| [C](#c)  | [I](#i) | [O](#o) | [U](#u) | 
| [D](#d)  | [J](#j) | [P](#p) | [V](#v) | 
| [E](#e)  | [K](#k) | [Q](#q) | [W](#w) | 
| [F](#f)  | [L](#l) | [R](#r) | [X](#x) | 
--------------------

Reply:
Yes, that's good I will do that @Vyvy-vi